b9lab-content/4-running-a-chain/04-node-api-and-cli.md

```diff
@@ -1 +1,464 @@
+      },
+      "signing_infos": []
+    },
+    "staking": {
+      "delegations": [],
+      "exported": false,
+      "last_total_power": "0",
+      "last_validator_powers": [],
+      "params": {
+        "bond_denom": "stake",
+        "historical_entries": 10000,
+        "max_entries": 7,
+        "max_validators": 100,
+        "unbonding_time": "1814400s"
+      },
+      "redelegations": [],
+      "unbonding_delegations": [],
+      "validators": []
+    },
+    "upgrade": {},
+    "vesting": {}
+  },
+  "chain_id": "test-chain-rT4wZY",
+  "gentxs_dir": "",
+  "moniker": "demo",
+  "node_id": "cf6bff39bb84da39d214138ebba8bcba4ccb848d"
+}
+```
+
+</ExpansionPanel>
+
+In the output, you can find your `chain_id`, which here happens to be `test-chain-rT4wZY`. Make a note of the one you have as you will need it later on to determine the chain id, by passing it to simapp via the flag `--chain-id`.
+
+You can inspect the initial configuration with:
+
+```bash
+$ cat ~/.simapp/config/genesis.json
+```
+
+## Prepare Your Account
+
+You can also inspect your keys, as you will need them. These are held in the backend keyring, which, by default, is that of the operating system:
+
+```bash
+$ ./simd keys list
+[]
+```
+
+As you might have expected, you do not have any keys yet. Let's remedy that and add a new key:
+
+```bash
+$ ./simd keys add b9lab
+
+- name: b9lab
+  type: local
+  address: cosmos1nw793j9xvdzl2uc9ly8fas5tcfwfetercpdfqq
+  pubkey: '{"@type":"/cosmos.crypto.secp256k1.PubKey","key":"A6TrsRO/OH91fAEFLohw7RwFB832NRsRWhQvE2t8cfLK"}'
+  mnemonic: ""
+
+
+**Important** write this mnemonic phrase in a safe place.
+It is the only way to recover your account if you ever forget your password.
+
+ivory uniform actual spot floor vessel monster rose yellow noise smile odor veteran human reason miss stadium phrase assault puzzle sentence approve coral apology
+```
+
+You can see the mnemonic at the end of the output:
+```bash
+ivory uniform actual spot floor vessel monster rose yellow noise smile odor veteran human reason miss stadium phrase assault puzzle sentence approve coral apology
+```
+
+This sequence of words is a mnemonic that you can use to recover your public and private keys. In a production setting, the mnemonic must be stored in a reliable and confidential fashion such as a key-management infrastructure.
+
+Confirm that the key has been added:
+
+```bash
+$ ./simd keys list
+```
+
+or with:
+
+```bash
+$ ./simd keys show b9lab
+```
+
+## Make Yourself a Proper Validator
+
+As you know by now, a Cosmos SDK blockchain relies on identified validators to produce blocks. Validators are added and removed through an on-chain governance process, i.e. via transactions. Initially, there is no validator to generate any block in which such a transaction might be added. You are in a catch-22 situation. Your initialized, but unstarted, chain needs a genesis account and validator for bootstrapping purposes.
```

Comment:
As far as I'm aware validators are not added or removed via a governance process, just automatically based on voting power. @xavierlepretre @rob-Hitchens 

```suggestion
As you know by now, a Cosmos SDK blockchain relies on identified validators to produce blocks. Initially, there is no validator to generate blocks. You are in a catch-22 situation. Your initialized and unstarted chain needs a genesis account and validator for bootstrapping purposes.
```